### PR TITLE
refactor(instr-graphql): use exported strings for attributes

### PR DESCRIPTION
**Pull request by david-luna**
## Which problem is this PR solving?

- Updates #2025 

## Short description of the changes

- Update @opentelemetry/semantic-conventions to ^1.22
- Replace `SemanticAttributes.*` with exported strings `SEMATTRS_*` in test files

Note: I did not add the "semantic conventions" section in the readme file since this instrumentation is using it only in tests but not for span attribs

### CODE REVIEW

**Review by blumamir — APPROVED** [review]
LGTM

Should we add some note in the README about this package not implementing any semantic convention at the moment?

**Comment by david-luna** [comment]
> LGTM
> 
> Should we add some note in the README about this package not implementing any semantic convention at the moment?

I thought the absence of the section in the README was enough but I based on your comment I guess it would be better that all packages have a section about semconv

**Comment by blumamir** [comment]
> > LGTM
> > Should we add some note in the README about this package not implementing any semantic convention at the moment?
> 
> I thought the absence of the section in the README was enough but I based on your comment I guess it would be better that all packages have a section about semconv

That's my personal opinion 😄 Feel free to add one or resolve if you prefer the READMEs not to state it in this case

**Comment by JamieDanielson** [comment]
> > > LGTM
> > > Should we add some note in the README about this package not implementing any semantic convention at the moment?
> > 
> > 
> > I thought the absence of the section in the README was enough but I based on your comment I guess it would be better that all packages have a section about semconv
> 
> That's my personal opinion 😄 Feel free to add one or resolve if you prefer the READMEs not to state it in this case

Oh interesting! It looks like graphql semantic conventions do exist, and were recently [added to the registry in semantic-conventions repo](https://github.com/open-telemetry/semantic-conventions/pull/882), though do not appear to exist in the schema itself yet unless I am missing it.

Also seeing there was an [issue opened (closed for stale)](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1612) referencing unexpected attributes. So this does seem like something that we explicitly should state that we are not using semantic conventions here for graphql attributes... and a new issue should perhaps be opened to review that. I'm not clear on whether this would be included in the new generated semantic conventions if it doesn't exist in the schema yaml files, but it is something that appears to have expected conventions at this point.

**Review by JamieDanielson — APPROVED** [review]
On #2182 I tried out this wording; if we like it we can use it here.

```markdown
## Semantic Conventions

This package does not currently generate any attributes from semantic conventions.
```

Also as far as the other attributes that do seem like they should be pulled from semantic conventions and possibly updated as well... let's deal with that separately. This is strictly documenting the current state.

**Comment by david-luna** [comment]
@JamieDanielson I've added your suggestion. Thanks :)

File: plugins/node/opentelemetry-instrumentation-graphql/package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
   "devDependencies": {
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/sdk-trace-base": "^1.8.0",
-    "@opentelemetry/semantic-conventions": "^1.3.1",
+    "@opentelemetry/semantic-conventions": "^1.22.1",
```
Comment (trentm) [inline plugins/node/opentelemetry-instrumentation-graphql/package.json:49]: This should be:

```suggestion
    "@opentelemetry/semantic-conventions": "^1.22.0",
```

and then package-lock updated. There isn't a 1.22.1 release.

Comment (david-luna) [reply plugins/node/opentelemetry-instrumentation-graphql/package.json]: done!

**Review by trentm — APPROVED** [review]

**Pull request merged**